builder/FindGlobals.cmake

```diff
@@ -92,8 +92,8 @@ if( Linux OR Darwin )
     set(no_warnings "-Wno-deprecated-declarations -Wno-unknown-pragmas -Wno-unused")
   endif()
 
-  set(CMAKE_C_FLAGS "-pipe -fPIC")
-  set(CMAKE_CXX_FLAGS "-pipe -fPIC")
+  set(CMAKE_C_FLAGS "-pipe -fPIC -Wnon-virtual-dtor")
```

Comment:
I agree that this is much more appropriate place for these flags than DEBUG/RELEASE flags below. However, could you, please, make 1 more enhancement? Add:
`set(warnings "-Wformat -Wformat-security -Wnon-virtual-dtor")`
right before, similar way we have 'no_warnings'. And eventually eliminate these flags below in DEBUG/RELEASE.

Reply:
done